taverna-cwl-activity-ui/pom.xml

```diff
@@ -65,27 +60,45 @@
 		</dependency>
 		<!-- =====================================These Dependencies were added 
 			later !!! =============================================================== -->
+
 		<dependency>
-			<groupId>org.yaml</groupId>
-			<artifactId>snakeyaml</artifactId>
-			<version>1.17</version>
+			<groupId>org.apache.taverna.commonactivities</groupId>
+			<artifactId>taverna-cwl-utilities</artifactId>
+			<version>${project.version}</version>
+			<type>bundle</type>
 		</dependency>
 		<dependency>
+			<artifactId>taverna-cwl-activity</artifactId>
+			<groupId>${project.groupId}</groupId>
+			<version>${project.version}</version>
+		</dependency>
+			
+		<dependency>
 			<groupId>net.sf.taverna.t2.ui-api</groupId>
 			<artifactId>activity-palette-api</artifactId>
 			<version>${t2.ui.api.version}</version>
 		</dependency>
-
 		<dependency>
-			<artifactId>taverna-cwl-activity</artifactId>
-			<groupId>${project.groupId}</groupId>
-			<version>${project.version}</version>
+			<groupId>net.sf.taverna.t2.ui-api</groupId>
+			<artifactId>menu-api</artifactId>
+			<version>${t2.ui.api.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>net.sf.taverna.t2.ui-api</groupId>
+			<artifactId>contextual-views-api</artifactId>
+			<version>${t2.ui.api.version}</version>
 		</dependency>
 		<dependency>
+			<groupId>net.sf.taverna.t2.ui-impl</groupId>
+			<artifactId>activity-tools</artifactId>
+			<version>${t2.ui.impl.version}</version>
+		</dependency>
+		
+		<dependency>
 			<groupId>net.sf.taverna.t2.taverna-workbench</groupId>
 			<artifactId>workbench-dev</artifactId>
 			<version>${t2.workbench.version}</version>
-			<scope>test</scope>
```

Comment:
`workbench-dev` must be commented out as it does not exist in Taverna 3 :-(  -- and anyway it should stay in `<scope>test`